app/jobs/scraper/washington_job.rb

```diff
@@ -0,0 +1,56 @@
+class Scraper::WashingtonJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    url_base = "https://secure.lni.wa.gov/arts-public/"
+    browser = Watir::Browser.new :chrome,
+      options: {args: %w[--headless --no-sandbox --disable-dev-shm-usage --disable-gpu]}
+    browser.goto(url_base + "#/program-search")
+    js_doc = browser.element(css: "div.lni-u-flex.lni-u-flex-wrap.lni-u-items-end").wait_until(&:present?)
+    js_doc.button(text: "Search").click
+
+    next_button = browser.a(aria_label: "Next").wait_until(&:present?)
+    program_ids = []
+
+    loop do
+      find_table = browser.element(css: "tbody").wait_until(&:present?)
+      begin
+        attempts ||= 0
+        table = Nokogiri::HTML(find_table.inner_html)
+      rescue Watir::Wait::TimeoutError
+        attempts += 1
+        sleep 60
+        retry if attempts <= 5
+      end
```

Comment:
Sorry I didn't catch this before -- now with the retry added, you will need to wrap all of the lines (26-33) below also in the `begin` block, otherwise, if this times out 5 times, then it will just continue on to line 26, but there will be no `table` object and it will error.

Reply:
And probably it would be good to report an error if it does timeout, so at least we are alerted if the script doesn't run.

```
rescue Watir::Wait::TimeoutError => e
  attempts += 1
  if attempts <= 5
    sleep 60
    retry
  else
    Rails.error.report(e)
  end
end
```